### ISSUE TITLE
testing-library/prefer-find-by should fix only if RTL getter is called on MemberExpression

### ISSUE THREAD

**Issue by RIP21** [testing-library/eslint-plugin-jest-dom#60]
**Problem description:**

If fix sees such code

```typescript
const { getByText } = render(<Whatever />);

const button = await waitForElement(() => getByText(ASSIGN_TM_TO_ME_SELECTOR));
```

It fixes it this way

```typescript
const { getByText } = render(<Whatever />);

const button = await findByText(ASSIGN_TM_TO_ME_SELECTOR);
```

This means the test will fail as `findByText` is undefined. Hence fix is destructive

**Suggested solution:**

In order to fix this, it will be nice to only apply fixes in case of RTL helper getter function called on other `MemberExpression`
e.g. `screen` typical usage or

```typescript
const result = render();
const button = await waitForElement(() =>
  result.getByText(ASSIGN_TM_TO_ME_SELECTOR)
);
```

or if a helper is imported by the file e.g.

```typescript
//                Here is the import
import { render, getByText, waitForElement } from "@testing-library/react";

render(<Whatever />);

const button = await waitForElement(() => getByText(ASSIGN_TM_TO_ME_SELECTOR));
```
Otherwise fix should be ignored

**Comment by RIP21** [comment]
Just noticed that it's not related to `jest-dom` sorry for that! :)
